app/src/main/java/ch/epfl/sdp/blindly/database/UserRepository.kt

```diff
@@ -50,6 +50,30 @@ class UserRepository @Inject constructor(
         return refreshUser(uid)
     }
 
+    /**
+     * Removes another liked or matched user from current user.
+     *
+     * @param field field to remove a User (either from LIKES or MATCHES)
+     * @param userUid current user's UID
+     * @param matchUid matched user's UID
+     */
+    suspend fun removeMatchFromAUser(field: String, userUid: String, matchUid:String) {
```

Comment:
I went through your whole pull request and have not found anything to comment on... x)
So I don't know, maybe for coherence you should rename the argument userUid to userId and matchId I guess....

Reply:
Sure I can do that :D